Elections/2025/Candidates.md

```diff
@@ -0,0 +1,30 @@
+# 2025 OpenFeature Governance Committee Candidates
+
+In February 2025 we will hold the election with 4 positions to be elected from the current Governance committee and 3 members from the community.
+The 4 members that are elected from the current governance board will have a one year term and the 3 elected from the community will have a two year term.
```

Comment:
There are only 4 total seats open. This should be rephrased to make that more clear.

Reply:
OK I've clarified here